test/cpp/test_status_common.h

```diff
@@ -0,0 +1,271 @@
+#ifndef XLA_TEST_CPP_TEST_STATUS_COMMON_H_
+#define XLA_TEST_CPP_TEST_STATUS_COMMON_H_
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "torch_xla/csrc/runtime/env_vars.h"
+#include "torch_xla/csrc/status.h"
+
+namespace torch_xla {
+
+// Enum to control whether C++ error context is shown in status messages
+enum class CppErrorContextMode {
+  SHOW,
```

Comment:
Per https://google.github.io/styleguide/cppguide.html#Enumerator_Names, this should be `kShow`.

Would be great to get familiar with the Google C++ style guide.